server/views/pages/report.njk

```diff
@@ -0,0 +1,133 @@
+{% extends "../partials/layout.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% set pageTitle = applicationName + " - Report" %}
+{% set mainClasses = "app-container govuk-body" %}
+{% block content %}
+  <h1>Report '{{ report.title }}'</h1>
+
+  {{ govukTable({
+  caption: "Report details",
+  captionClasses: "govuk-table__caption--m",
+  firstCellIsHeader: true,
+  rows: [
+    [
+      {
+        text: "Type"
+      },
+      {
+        text: report.type
+      }
+    ],
+    [
+      {
+        text: "Title"
+      },
+      {
+        text: report.title
+      }
+    ],
+    [
+      {
+        text: "Reference"
+      },
+      {
+        text: report.reportReference
+      }
+    ],
+    [
+      {
+        text: "Status"
+      },
+      {
+        text: report.status
+      }
+    ],
+    [
+      {
+        text: "When reported"
+      },
+      {
+        text: report.reportedAt
+      }
+    ],
+    [
+      {
+        text: "Reported by"
+      },
+      {
+        text: report.reportedBy
+      }
+    ],
+    [
+      {
+        text: "Description"
+      },
+      {
+        text: report.description
+      }
+    ]
+  ]
+  }) }}
+
+  <h2>Prisoners involved</h2>
+  {%- if report.prisonersInvolved.length > 0 -%}
+    <ul>
+      {%- for prisonerInvolved in report.prisonersInvolved -%}
+        <li>{{ prisonerInvolved.prisonerNumber }} ({{ prisonerInvolved.prisonerRole }})</li>
```

Comment:
should we include outcome? might help to spot whether it's been used in the past

Reply:
Maybe, bear in mind these pages don't show everything-everything.

I've tweaked the page to show the prisoner outcomes as well, although I couldn't quickly think of any clever way of displaying this (but also it doesn't matter at this stage)

[Show prisoner involvement outcome](https://github.com/ministryofjustice/hmpps-incident-reporting/pull/131/commits/88b71990fd36c82bd5eae5f7f1574541ecd6eb67)